axiom/nr_errors_private.h

```diff
@@ -25,6 +25,11 @@ struct _nr_error_t {
   int priority;          /* Error priority - lowest to highest */
   char* message;         /* Error message */
   char* klass;           /* Error class */
+  char* error_file;      /* Error file */
+  int error_line;        /* Error line */
+  char* error_context;   /* Error context */
+  int error_no;          /* Error number */
+  int option;            /* Error option */
```

Comment:
So am I correct in that the `option` variable indicates whether the other user-provided variables are populated? Could we not just check, for example, `error_file` for null behind an API like `nr_error_has_user_attributes` rather than having this confusing variable? As far as I can tell, these attributes are either all-or-nothing.

Reply:
Agree, the `option` variable isn't necessary and I have removed it: [9f3e7e6](https://github.com/newrelic/newrelic-php-agent/pull/942/commits/9f3e7e652eeeaef0e19c45a1aacbd5ba6891d161)